conf.yaml

```diff
@@ -891,7 +891,7 @@ contents:
             prefix:     en/cloud-on-k8s
             tags:       Kubernetes/Reference
             subject:    ECK
-            current:    1.8
+            current:    1.9
```

Comment:
We'll need to also add `1.9` to the `branches` list below here in order for the 1.9 version to be available.

Reply:
Oh 🤦🏻 . I see you did that in #2301